data_model.mdd

```diff
@@ -355,13 +358,27 @@
         bool enable_notification "Parental leave or career breaks require notifications if they are within 30days"
     }
     SupportRecord {
-        int id
+        int id PK
         int trainee_id FK
-        date date "Support Record Date Entered onto DB, Null: True"
+        int case_type_id FK "Support Issue Type, Null: True"
+        date open_date "Support Record Date Entered onto DB, Null: True"
         string(MAX) comments "Support Description, Null: True"
+        int support_status_id FK "Support Status, Null: True"
```

Comment:
Note: perhaps we should also enforce this?

Reply:
So this is "fun"

Since I am extending existing `SupportRecord` - import Excel data won't have such a column

my idea was to leave it nullable for import, after which a dev would go and manually add respective status to the db (via OutSystems) after consulting with school 

now that I think of it - we can just default all imported records to 'Closed' status, which can be edited by users later on

TLDR - removing nullability